code/__HELPERS/atoms.dm

```diff
@@ -313,6 +313,8 @@ rough example of the "cone" made by the 3 dirs checked
 ///A do nothing proc
 /proc/pass(...)
 	return
+///A do nothing proc
+/proc/noop(...)
```

Comment:
dont keep both. pick one.

Reply:
```suggestion
///A do nothing proc
/proc/noop(...)
```
Less gross, `pass()` is used in unit test code.